invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less

```diff
@@ -247,3 +247,24 @@ svg:not(:root) {
 span.home-inline{
   white-space: nowrap;
 }
+
+.test-instance-side-sticky{
```

Comment:
try using semantic ui - for sticky classes. https://semantic-ui.com/modules/sticky.html

Reply:
thank for the hint